_guides/how-we-do-development.md

```diff
@@ -59,8 +57,7 @@ for.
 
 For example, while working on a card with ID `123` about generating a report:
 
-* Good: `feature/123-generate-report`, `123/generate-report`,
-  `123-generate-report`
+* Good: `123/generate-report`, `123-generate-report,` `feature/123-generate-report`
```

Comment:
Moved `feature/123` down the list as it feels like something we're doing less of.